ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7.yaml

```diff
@@ -293,6 +293,50 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: openshift-e2e-aws
+- as: e2e-mesh-ocp-47-continuous
+  cron: 0 0 * * 0
```

Comment:
Any reason this cron is different from the others? All others are `0 */12 * * 1-5`

Reply:
No, there is no reason. I just copied it from aws-ovn yaml and it has different cron schedule.
Updated.